code/datums/datumvars.dm

```diff
@@ -779,6 +779,7 @@ function loadPage(list) {
 			M.teleport_to(T)
 		else
 			A.forceMove(T)
+		log_admin("[key_name(usr)] has teleported [A] at ([T.x], [T.y], [T.z]).")
```

Comment:
```suggestion
		log_admin("[key_name(usr)] has teleported [A] from [formatLocation(AT)] to [formatLocation(T)].")
```

Reply:
This proc gives the area name and the coordinates of the destination turf, please use it instead.